wandb/sdk/wandb_run.py

```diff
@@ -2154,7 +2163,7 @@ def _create_artifact_job(
         ):
             return None
         artifact_client_id = self._code_artifact_info.get("client_id")
-        name = f"job-{self._code_artifact_info['name']}"
+        name = self._get_job_name(f"job-{self._code_artifact_info['name']}")
```

Comment:
why you don't use the `wandb.util.make_artifact_name_safe` here?
and if you always need it maybe it makes sense to move it into the `self._get_job_name` method?

Reply:
The code artifact name is already safe since it is an artifact.
I can move it though, good idea

Reply:
got ya! Thanks!